pkg_defs/bep_pcb_check/build.sh

```diff
@@ -0,0 +1,2 @@
+export SKA=/proj/sot/ska
+pip install --no-deps --verbose --no-binary :all: --no-index .
```

Comment:
We're pushing to update the installs to use

```
python setup.py install --single-version-externally-managed --record=record.txt
```
Will probably move those into the meta.yaml at some point too, but at this point it would probably be good just to get the `python setup.py` transition.  Probably also don't need to define SKA.